GeniusLyrics.js

```diff
@@ -2069,8 +2097,14 @@ Genius:  ${originalUrl}
       }
     }
     iframe.src = custom.emptyURL + '#html:post'
-
     custom.setFrameDimensions(container, iframe, bar)
+    song = `${song}` // ensure typeof song === 'string'
```

Comment:
There is an error `TypeError: song.result is undefined` in line 497 https://github.com/cvzi/genius-lyrics-userscript/blob/24c8cc81df42f4ffa44bba55ed4edb0ebb752113/GeniusLyrics.js#L497 probably because of this change.

Reply:
Resolved